### DISCUSSION TITLE
How do I add a new node to a cluster using Docker Compose？

### DISCUSSION THREAD

**Discussion by gg381534551** [milvus-io/milvus#15768] [Q&A and General discussion]
I use Docker Compose to deploy distributed Milvus, but it is not clear how to add another distributed Milvus to the cluster. The content of the document is to use Kubernetes for capacity expansion. I haven't found Docker. I hope you can help me.

**Comment by xiaofan-luan** [comment]
docker compose can not add more node, it should be just used as test.
To deploy distributed cluster you should use K8s Operator.

**Reply by gg381534551** [reply]
谢谢！ I'm Chinese！
